README.md

```diff
@@ -1 +1 @@
-testing semantic versioning 4.10.2
+testing semantic versioning 4.10.7
```

Comment:
- The version number has been updated from 4.10.2 to 4.10.7. No further comments.